src/pybind/mgr/cephadm/serve.py

```diff
@@ -1429,8 +1429,24 @@ async def _create_daemon(self,
                         config_blobs=daemon_spec.final_config,
                     ).dump_json_str(),
                     use_current_daemon_image=reconfig,
+                    error_ok=True
```

Comment:
this change is going to require updating a whole bunch of unit tests, hence the make check failure

Reply:
@adk3798 plz, can you give some context about what the purpose of this variable? form its name it's not clear to me what are we capturing here. 

BTW: I can't see where the variable is set to another value.

Reply:
it defaults to `False` in both the `_run_cephadm` and `_run_cephadm_json` function signatures. If it's `True` an exception will be thrown if we get a nonzero rc from a cephadm command we run. With it `False`, it will return the rc, stdout, and stderr even in the case of a nonzero rc. We need it to not throw the exception in order for us to examine the rc from the call and decide whether it's the case this PR is about that needs special handling.